services/external_sources/worldbank.py

```diff
@@ -27,19 +27,24 @@ def worldbank_search(query, owner, limit=5, prev=0):
     logger.debug(f"Searching worldbank for query: {query}")
     res = []
     try:
-        search_meta = wb.search2(q=query)
+        if query == "":
+            search_meta = wb.search2(q="woman")
```

Comment:
Why a hardcoded search term for "woman"? Is this per design?

If we keep it, make it a constant named something like DEFAULT_SEARCH_TERM for clarity

Reply:
It's not per design. It's just a default search term. We can always update it to something more general which also returns values from worldbank.